modules/core/example/ts/avaxc-test.ts

```diff
@@ -0,0 +1,44 @@
+import { BitGo } from 'bitgo';
```

Comment:
Can we make this file more like a template for clients like other examples in this dir? (rather than template for bitgo engineers)
- custom env is not applicable to our clients, let's leave it as test
- leave access token and wallet id blank
- remove commented lines
- leave wallet passphrase and address empty 
- put a couple of comments to clean up

Reply:
sure